ope/sampling/metric.py

```diff
@@ -0,0 +1,17 @@
+from typing import Callable, Dict, Union
+
+import pandas as pd
+
+
+def confidence_interval(p_value: float) -> Callable:
+    """Returns a callable which evaluates a confidence interval on a series."""
+
+    def get_confidence_interval(series: pd.Series) -> Dict[str, Union[str, float]]:
+        mean = series.mean()
+        standard_dev = series.std()
+        return {
+            "name": f"confidence_interval_{p_value}",
+            "value": [mean - 1.96 * standard_dev, mean + 1.96 * standard_dev],
```

Comment:
nit: pull out 1.96 as a constant somewhere

Reply:
Apologies, no idea how the magic number version is still in here, this was the quick and dirty one I was using for early testing. Had a version that uses scipy.norm if you're comfortable having scipy as a dependency (It's a pretty stable dependency of scikit-learn so it shouldn't pull the rug out from under anything for now unless you want to feature-gate the scikit-learn stuff at any point).

Reply:
No problem. Prefer to leave Scipy out for now!